apps/jobs/serializers.py

```diff
@@ -163,6 +165,27 @@ def get_participant_team_members(self, obj):
             )
         )
 
+    def get_participant_team_members_affiliations(self, obj):
+        try:
+            participant_team = ParticipantTeam.objects.get(
+                team_name=obj.participant_team.team_name
+            )
+        except ParticipantTeam.DoesNotExist:
+            return "Participant team does not exist"
+
+        participant_ids = Participant.objects.filter(
+            team=participant_team
+        ).values_list("user_id", flat=True)
+        users = User.objects.filter(id__in=participant_ids)
+        participant_team_members_affiliation = list()
+        for user in users:
+            if user.profile.affiliation == "":
+                affiliation = "None"
```

Comment:
Instead of having a special case handling we can straight away append empty strings to `participant_team_members_affiliation`. It will result in a valid csv like this:
```csv
id,Team Name,Team Members Affiliation,Challenge Phase
2,sdf,,Linda Phase
1,West Elizabeth Participant Team,aff name,Linda Phase
```

Which should be good. or Is there any specific reason other than this to Inject "None" string inplace of when affiliation field's are empty. 
If we don't have to do the above special handling, You can simplify the code too to 
```python
return [user.profile.affiliation for user in users]
```

Reply:
@galipremsagar Actually I did this way cause if there are more than one participants in a team then it will be easy to know which participant's affiliation is `None` and whose affiliation is provided.

Reply:
You mean for filtering in google docs & excel? If so then lets get rid of Explicit `"None"` string insertion. We can filter empty fields in both of them like this:
<img width="282" alt="Screenshot 2019-08-17 at 4 32 50 PM" src="https://user-images.githubusercontent.com/11664259/63210499-b14ea380-c10c-11e9-929b-818401c065fc.png">

Reply:
okay, got your point @galipremsagar , thanks :+1: Made the changes, please review